packages/razzle/config/createConfig.js

```diff
@@ -237,15 +214,16 @@ module.exports = (
       config.watch = true;
       config.entry.unshift('webpack/hot/poll?300');
 
-      config.plugins = [
-        ...config.plugins,
-        // Add hot module replacement
-        new webpack.HotModuleReplacementPlugin(),
-        // Supress errors to console (we use our own logger)
-        new webpack.NoEmitOnErrorsPlugin(),
-        // Automatically start the server when we are done compiling
-        new StartServerPlugin('server.js'),
-      ];
+      config.plugins.push(
+        ...[
```

Comment:
why put it in an array and destructure it again? why not pass it as the contents as arguments straight away?

Reply:
@jariz I'm not quite sure I understand your question. from what I understand of spread operator this is the same as

```js
Array.prototype.push.apply(config.plugins, [<a_bunch_of_plugins])
```
which is more performant than `.concat` and in my opinion more concise than the pattern of mutating by redefining to the same variable that you are concat'ing into...if that makes any sense 

```js
config.plugins = [
  ...config.plugins,
```